sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -995,6 +990,126 @@ public void captureSession(final @NotNull Session session, final @Nullable Hint
     return sentryId;
   }
 
+  /**
+   * Captures the feedback.
+   *
+   * @param feedback The feedback to send.
+   * @param hint An optional hint to be applied to the event.
+   * @param scope An optional scope to be applied to the event.
+   * @return The Id (SentryId object) of the event
+   */
+  @Override
+  public @NotNull SentryId captureFeedback(
+      final @NotNull Feedback feedback, @Nullable Hint hint, final @NotNull IScope scope) {
+    SentryEvent event = new SentryEvent();
+    event.getContexts().setFeedback(feedback);
+
+    if (hint == null) {
+      hint = new Hint();
+    }
+
+    if (feedback.getUrl() == null) {
+      feedback.setUrl(scope.getScreen());
+    }
+
+    options.getLogger().log(SentryLevel.DEBUG, "Capturing feedback: %s", event.getEventId());
+
+    if (shouldApplyScopeData(event, hint)) {
+      // Event has already passed through here before it was cached
+      // Going through again could be reading data that is no longer relevant
+      // i.e proguard id, app version, threads
+      event = applyFeedbackScope(event, scope, hint);
+
+      if (event == null) {
+        options.getLogger().log(SentryLevel.DEBUG, "Feedback was dropped by applyScope");
+        return SentryId.EMPTY_ID;
+      }
+    }
+
+    event = processFeedbackEvent(event, hint, options.getEventProcessors());
+
+    if (event != null) {
+      event = executeBeforeSendFeedback(event, hint);
+
+      if (event == null) {
+        options.getLogger().log(SentryLevel.DEBUG, "Event was dropped by beforeSend");
+        options
+            .getClientReportRecorder()
+            .recordLostEvent(DiscardReason.BEFORE_SEND, DataCategory.Feedback);
+      }
+    }
+
+    if (event == null) {
+      return SentryId.EMPTY_ID;
+    }
+
+    SentryId sentryId = SentryId.EMPTY_ID;
+    if (event.getEventId() != null) {
+      sentryId = event.getEventId();
+    }
+
+    final boolean isBackfillable = HintUtils.hasType(hint, Backfillable.class);
```

Comment:
i think we don't need this check because the FeedbackEvent will never be `Backfillable` (it's only for ANRv2 events basically, which go through `SentryClient.captureEvent`)